packages/react-router-dev/vite/plugin.ts

```diff
@@ -756,7 +756,7 @@ export const reactRouterVitePlugin: ReactRouterVitePlugin = () => {
         ctx.reactRouterConfig.future.unstable_viteEnvironmentApi &&
         viteCommand === "serve"
           ? `
-              export const getCriticalCss = ({ pathname }) => {
+              export const unstable_getCriticalCss = ({ pathname }) => {
```

Comment:
Seeing this API in a new light as part of this PR made me realise this should be marked as unstable. It was only ever defined when setting the `unstable_viteEnvironmentApi` flag.